sys/net/routing/rpl/Makefile

```diff
@@ -1,8 +1,16 @@
-MODE = $(shell grep "RPL_DEFAULT_MOP RPL_NON_STORING_MODE" ../../include/rpl/rpl_config.h)
-ifneq (,$(MODE))
-	DIRS += rpl_nonstoring
+MODE = $(shell grep "\#define $(RPL_MOP)" ../../include/rpl/rpl_config.h | sed 's/\#define \s*//g')
```

Comment:
I'm also not a big fan of grep the options from the header file, but this way the user/developer can only apply valid values on command line.
For instance if we, for whatever reason, change `RPL_MOP_STORING_MODE_NO_MC` to `0xdeadbeef`, the user/developer cannot accidentally set the `RPL_MOP` to `0xfeedbeef`